### PR TITLE
Basic project scaffold.

### PR DESCRIPTION
**Pull request by emartinez-usgs**
Here is an empty project scaffold? Thoughts? I intend to link to several of these files as examples from the "Getting Started" documentation I am writing up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+include only the proper disclaimer.
+
+
+### ./LICENSE.md
+
+This file contains the license under which the project source code is released.
+The abscence of a license does not automatically make something open-source and
+therefor this file (or similar documentation) is **required**. Typically
+government work falls under "Public Domain", however [CC0-1.0][3] may be used for
+international recognition.
```
Comment (mguy-usgs) [inline README.md]: I recommend adding that licenses of any 3rd party libraries/code dependencies should be listed here as well

File: README.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+Example Project
+===============
+
+As part of the usgs/best-practices repository, this branch makes an effort to
+provide a starting point for new or empty projects. Recognizing each project
+and language is different and may come with its own set of best practices,
+these are merely reasonable guidelines from which to start.
```
Comment (mguy-usgs) [inline README.md]: maybe redundant, but I would recommend again mentioning here this example is provided as guidance but USGS policy takes precedence.

Comment (emartinez-usgs) [inline README.md]: Added clarifying statement in e09f04a

File: README.md (hunk 1 above)
Comment (emartinez-usgs) [inline README.md]: Added clarification in 25c4646

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+Contributing
+============
+
+Contributions are welcome from the community. Questions can be asked on the
+[issues page][1]. Before creating a new issue, please atake a moment to search
```
Comment (jmfee-usgs) [inline CONTRIBUTING.md]: `atake` => `take`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+>       the repository even though it is otherwise empty. If you clone this
+>       branch as a starting point for an actual repository, you will want to
+>       delete this "./src/.gitignore" file.
+
+
+### ./.gitignore
+
+This file informs Git which files should not be monitored for version control.
+These are typically system or generated files (such as compiled binaries). You
+should update this file with the build artificats you wish to exclude from
```
Comment (jmfee-usgs) [inline README.md]: `artificats` => `artifacts`

**Pull request merged**